tests/e2e/clusterreset/clusterreset_test.go

```diff
@@ -38,6 +39,11 @@ var (
 	agentNodeNames  []string
 )
 
+var _ = ReportAfterEach(func(specReport SpecReport) {
+	status := e2e.GenReport(specReport)
+	fmt.Printf("%s", status)
+})
+
```

Comment:
You can consolidate all of these even more.
```suggestion
var _ = ReportAfterEach(e2e.GenReport)

```